src/main/java/frc/robot/commands/IntakeCommand.java

```diff
@@ -0,0 +1,75 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Shooter;
+import frc.robot.Constants;
+
+import static frc.robot.Constants.OPERATOR_CONTROLLER;
+
+
+public class IntakeCommand extends Command {
+
+    private final Shooter shooter;
+    private final XboxController controller;
+    private double rightTrigger;
```

Comment:
Dont need this because the speed of the intaking command should be a constant value